encoder-brave/src/main/java/zipkin2/reporter/otel/brave/InstrumentationScope.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.reporter.otel.brave;
+
+import java.util.Objects;
+
+/**
+ * OpenTelemetry InstrumentationScope
```

Comment:
Let's add a little more to this
```suggestion
 * Instrumentation Scope indicates the application component that produced spans. This is more
 * specific than the local service name.
 * <p>
 * See https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope
 *
 * @see BraveScope#instrumentationScope
```